sample_files/pre-commit-13.0/.eslintrc.yml

```diff
@@ -5,56 +5,74 @@ env:
 parserOptions:
   ecmaVersion: 2017
 
-# Globals available in Odoo that shouldn't produce warnings
+# Globals available in Odoo that shouldn't produce errorings
 globals:
-  $: false
-  _: false
-  fuzzy: false
-  jQuery: false
-  moment: false
-  Promise: false
-  odoo: false
-  openerp: false
+  _: readonly
+  $: readonly
+  fuzzy: readonly
+  jQuery: readonly
+  moment: readonly
+  odoo: readonly
+  openerp: readonly
+  Promise: readonly
 
 # Styling is handled by Prettier, so we only need to enable AST rules;
 # see https://github.com/OCA/maintainer-quality-tools/pull/618#issuecomment-558576890
 rules:
+  accessor-pairs: warn
+  array-callback-return: warn
+  callback-return: warn
+  capitalized-comments:
+    - warn
+    - always
+    - ignoreConsecutiveComments: true
+      ignoreInlineComments: true
+  complexity:
+    - warn
+    - 15
```

Comment:
Perhaps 11 is a better value? See: https://github.com/eslintcc/eslintcc#complexity-ranks

Reply:
Humm I guess we'll have to see this in practice to judge, just by seeing the numbers I'm not very able to guess it.